README.rst

```diff
@@ -58,3 +79,6 @@ You can test your doc changes locally with::
     $ cd docs
     $ make html
     $ open _build/html/index.html
+
+
+[service-dev-wiki]: https://github.com/yola/wiki/wiki/Service-and-Client-guidelines#development
```

Comment:
Removed private wiki link in https://github.com/yola/yolapy/commit/f056fe46d4099f4c1886d064a846709d431f0792